application/src/ext/java/org/opentripplanner/ext/dataoverlay/GenericDataFile.java

```diff
@@ -26,7 +26,7 @@ public class GenericDataFile {
 
   private final String dataSource;
   private OffsetDateTime originDate;
-  private String error = null;
+  private String error;
```

Comment:
What do we think about removal of "default" values from member variables. I don't have a strong opinion on this but it's slightly less consistent if there is a false and true booleans next to each other and only one shows the value explicitly.

Reply:
I like it for `null` but for anything else, I find it less useful and - as you pointed out - inconsistent.